### DISCUSSION TITLE
id: cannot find name for group ID "99909997"

### DISCUSSION THREAD

**Discussion by aos92** [termux/termux-packages#12255] [Q&A] [answered]
When in "proot-distro" and install "archlinux" at the end I get the message id: cannot find name for group ID "99909997".  What do you think ?

****************************************

[*] Checking integrity, please wait...
[*] Extracting rootfs, please wait...
[*] Writing '/data/data/com.termux/files/usr/var/lib/proot-distro/installed-rootfs/debian/etc/profile.d/[termux-proot.sh](http://termux-proot.sh/)'...
[*] Writing resolv.conf file (NS [1.1.1.1/1.0.0.1](http://1.1.1.1/1.0.0.1))...
[*] Writing hosts file...
[*] Registering Android-specific UIDs and GIDs...
id: cannot find name for group ID [99909997](tel:99909997)
[*] Running distro-specific configuration steps...
gvfs-daemons set on hold.
udisks2 set on hold.
[*] Installation finished.

Now run 'proot-distro login archlinux' to log in.

****************************************
~ $ termux-info
Termux Variables:
TERMUX_API_VERSION=0.50.1
TERMUX_APK_RELEASE=F_DROID
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_APP_PID=23618
TERMUX_IS_DEBUGGABLE_BUILD=0
TERMUX_MAIN_PACKAGE_FORMAT=debian
TERMUX_VERSION=0.118.0
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://packages.termux.dev/apt/termux-main/ stable main
Updatable packages:
apt/stable 2.5.3 aarch64 [upgradable from: 2.5.2-3]
termux-tools version:
1.29.3
Android version:
11
Kernel build information:
Linux localhost 4.19.127+ #1 SMP PREEMPT Fri Sep 2 23:02:50 CST 2022 aarch64 Android
Device manufacturer:
OPPO
Device model:
CPH2269
~ $ id
uid=10680(u0_a680) gid=10680(u0_a680) groups=10680(u0_a680),3003(inet),9997(everybody),20680(u0_a680_cache),50680(all_a680),99909997

**Comment by orzomaxx** [comment, marked as answer]
This is ok. I have same problem on my Realme 7. It is caused by app cloning mechanism in your ROM

**Reply by aos92** [reply]
hmm.. 🤔 thanks.. very helpful  🙏☕☕
